src/libraries/Microsoft.PowerFx.Core/Parser/ParserExtensions.cs

```diff
@@ -10,9 +11,9 @@ namespace Microsoft.PowerFx.Core.Parser
     /// </summary>
     internal static class ParserExtensions
     {
-        internal static ParserOptions ToParserOptions(this TexlParser.Flags flags)
+        internal static ParserOptions ToParserOptions(this TexlParser.Flags flags, CultureInfo culture)
```

Comment:
>ToParserOptions [](http://example.com/codeflow?start=38&length=15)

This is only used in tests/Microsoft.PowerFx.Interpreter.Tests - it should be moved there, and not left in Microsoft.PowerFx.Core

Reply:
Created [https://github.com/microsoft/Power-Fx/issues/1312](https://github.com/microsoft/Power-Fx/issues/1312)